### PR TITLE
Issue 3028 - Federation settings UI fixes

### LINKED ISSUE #3028: Federation settings UI fixes
## Description
There are some problems pointed out by George with the federation settings form

### Tasks
- [x] Remove arrows from Lat and Long inputs
image

- [x] Incorrect font being used for input labels . Letter spacing and font weight seems incorrect.
image

- [x] Arrows for select inputs are incorrect.
image

- [ ] Border radius is incorrect for image on default view. Top right and bottom right border radius should be 0 once selected.
image

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #3028

#### Description
- Removed arrows from Lat and Long inputs;
- Fixed unit-labels typography;
- Dropdown menus use the correct arrow icon;
- Fixed order radius for image on default view

### CODE REVIEW

**Comment by Amantini1997** [comment]
https://issue-3028.dev.3drepo.io/

File: frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx [hunk 1]
```diff
@@ -48,7 +49,7 @@ export const FormSelect = ({
 					labelId={`${name}-label`}
 					id={name}
 					label={label}
-					{...otherProps}
+					className={className}
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx:52]: what happened here? `otherProps`  would also work for `className`

File: frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx [hunk 2]
```diff
@@ -16,8 +16,9 @@
  */
 
 import React from 'react';
-import { Select, InputLabel, FormControl, SelectProps } from '@material-ui/core';
+import { InputLabel, FormControl, SelectProps } from '@material-ui/core';
 import { Controller } from 'react-hook-form';
+import { Select } from '@controls/select';
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx:21]: We can get rid of  our Select control and just put those props as default props in the theme.

File: frontend/src/v5/ui/routes/dashboard/projects/federations/federationSettingsForm/federationSettingsForm.component.tsx [hunk 3]
```diff
@@ -209,14 +209,13 @@ export const FederationSettingsForm = ({ open, federation, onClose }: IFederatio
 				name="defaultView"
 				label={formatMessage({ id: 'federations.settings.form.defaultView', defaultMessage: 'Default View' })}
 			/>
-			<SectionTitle>GIS servey point</SectionTitle>
+			<SectionTitle>GIS survey point</SectionTitle>
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/dashboard/projects/federations/federationSettingsForm/federationSettingsForm.component.tsx:212]: Didn't notice this before. It should be localisable:

```suggestion
			<SectionTitle> 
				<FormattedMessage
					id="federations.settings.form.gisTitle"
					defaultMessage="GIS survey point"
				/>
			</SectionTitle>
```

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

**Comment by george-hadfield** [comment]
The spacing between the input titles and the input boxes are too close in the GIS Survey point section. The spacing should be consistent across all input boxes. Can you please make the spacing match the top half of the modal's boxes?

image

**Comment by george-hadfield** [comment]
The left padding beside the image on default view is too big. The top, bottom and left padding should be equal

image

**Review by george-hadfield — APPROVED** [review]

File: frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx (hunk 1 above)
Comment (Amantini1997) [reply frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx:52]: Fixed

File: frontend/src/v5/ui/routes/dashboard/projects/federations/federationSettingsForm/federationSettingsForm.component.tsx (hunk 3 above)
Comment (Amantini1997) [reply frontend/src/v5/ui/routes/dashboard/projects/federations/federationSettingsForm/federationSettingsForm.component.tsx:212]: Fixed

File: frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx (hunk 2 above)
Comment (Amantini1997) [reply frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx:21]: Done

**Comment by Amantini1997** [comment]
> The spacing between the input titles and the input boxes are too close in the GIS Survey point section. The spacing should be consistent across all input boxes. Can you please make the spacing match the top half of the modal's boxes?
> 
> image

Fixed

**Comment by Amantini1997** [comment]
> The left padding beside the image on default view is too big. The top, bottom and left padding should be equal
> 
> image

Fixed

**Review by george-hadfield — APPROVED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**